nodebuilder/header/header.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/celestiaorg/celestia-node/header"
 	"github.com/celestiaorg/celestia-node/header/p2p"
 	"github.com/celestiaorg/celestia-node/header/store"
-	"github.com/celestiaorg/celestia-node/params"
+	network "github.com/celestiaorg/celestia-node/nodebuilder/p2p"
```

Comment:
The overlapping of module pkgs and real libs will be a common thing. Should we agree on the pattern naming module imports something like `modp2p`? 

cc @distractedm1nd @renaynay

Reply:
Yes I think that's a good idea cc @distractedm1nd